src/modules/0.0.21/iasql_functions/iasql.ts

```diff
@@ -0,0 +1,301 @@
+// TODO: It seems like a lot of this logic could be migrated into the iasql_platform module and make
+// sense there. Need to think a bit more on that, but module manipulation that way could allow for
+// meta operations within the module code itself, if desirable.
```

Comment:
This TODO is being accomplished in this PR (when I wrote it, we didn't have a separate `iasql_functions` module).